src/rotp/ui/races/RacesUI.java

```diff
@@ -660,26 +659,70 @@ else if (otherDipGone)
                 }
             }
         }
-        public BufferedImage raceBackImg() {
-            if (raceBackImg == null)
-                initRaceBackImg();
-            return raceBackImg;
+        public BufferedImage raceBackImg(Empire emp) {
+             
+            if (emp.raceBackImage() == null )
+                initRaceBackImg(emp);            
+            return emp.raceBackImage();
+               
         }
-        private void initRaceBackImg() {
+        private void initRaceBackImg(Empire emp) {
             int w = s76;
             int h = s82;
-            raceBackImg = gc().createCompatibleImage(w, h);
-
+            emp.setRaceBackImage(gc().createCompatibleImage(w, h));
+            
             Point2D center = new Point2D.Float(w/2, h/2);
             float radius = s78;
             float[] dist = {0.0f, 0.1f, 0.5f, 1.0f};
-            Color[] colors = {raceCenterColor, raceCenterColor, raceEdgeColor, raceEdgeColor};
+            // gradient - method 1 (less flexible)
+//            Color[] colors = {
+//                emp.color().brighter().brighter(), 
+//                emp.color().brighter().brighter(), 
+//                emp.color().darker().darker(), 
+//                emp.color().darker().darker()
+//            };
+            // gradient - method 2 - blending colors (more flexible)
+            Color[] colors = {
+                blendColor(raceCenterColor,emp.color(), 0.6f), 
+                blendColor(raceCenterColor,emp.color(), 0.6f),
+                blendColor(raceEdgeColor,emp.color(), 0.6f),
+                blendColor(raceEdgeColor,emp.color(), 0.6f)
+            };
+            
+            
             RadialGradientPaint p = new RadialGradientPaint(center, radius, dist, colors);
-            Graphics2D g = (Graphics2D) raceBackImg.getGraphics();
+            Graphics2D g = (Graphics2D) emp.raceBackImage().getGraphics();
             g.setPaint(p);
             g.fillRect(0, 0, w, h);
             g.dispose();
         }
+        
+        private Color blendColor( Color c1, Color c2, float ratio ) {
+            if ( ratio > 1f ) ratio = 1f;
+            else if ( ratio < 0f ) ratio = 0f;
+            float iRatio = 1.0f - ratio;
+
+            int i1 = c1.getRGB();
+            int i2 = c2.getRGB();
+
+            int a1 = (i1 >> 24 & 0xff);
+            int r1 = ((i1 & 0xff0000) >> 16);
+            int g1 = ((i1 & 0xff00) >> 8);
+            int b1 = (i1 & 0xff);
+
+            int a2 = (i2 >> 24 & 0xff);
+            int r2 = ((i2 & 0xff0000) >> 16);
+            int g2 = ((i2 & 0xff00) >> 8);
+            int b2 = (i2 & 0xff);
+
+            int a = (int)((a1 * iRatio) + (a2 * ratio));
+            int r = (int)((r1 * iRatio) + (r2 * ratio));
+            int g = (int)((g1 * iRatio) + (g2 * ratio));
+            int b = (int)((b1 * iRatio) + (b2 * ratio));
+
+            return new Color( a << 24 | r << 16 | g << 8 | b );
+        }        
```

Comment:
Of course, if the "blending" option is used, this method really should not be here